src/styles/index.ts

```diff
@@ -4,6 +4,7 @@ import lodashClamp from 'lodash/clamp';
 import type {LineLayer} from 'react-map-gl';
 import type {Animated, ImageStyle, TextStyle, ViewStyle} from 'react-native';
 import {Platform, StyleSheet} from 'react-native';
+// eslint-disable-next-line no-restricted-imports -- will be removed in the future PR
```

Comment:
Initially I wanted to migrate animatable to reanimated in one PR but changes from AnimatedStep component become too big to migrate everything in one PR. [Here](https://github.com/Expensify/App/pull/54404/files) is the second PR that will be ready to review after this one is merged.